src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -47,93 +70,1257 @@ static ColliderCylinderInit D_80A9A100 = {
+    { 0x06009B64, 0.0f, 2.0f, 2.0f, ANIMMODE_ONCE, 0.0f },
+    { 0x060062DC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060062DC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -10.0f },
+    { 0x06005808, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -10.0f },
+    { 0x06007830, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06008178, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060065E0, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x0600879C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007FFC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060080B4, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060091AC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06006F9C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007064, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007120, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007F38, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007D94, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06006EE0, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060098EC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060090EC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x0600982C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06009274, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x060099A4, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06009028, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007E64, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06007454, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { 0x06008F6C, 0.0f, 1.0f, 1.0f, ANIMMODE_LOOP_PARTIAL, -8.0f },
+    { 0x06007D94, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
+    { 0x0600879C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
+    { 0x06006A60, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
+    { 0x06007830, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
+};
+
+static u8 sOsAnimeLookup[13][5] = {
+    /* ENKO_TYPE_CHILD_0    */ { 0x08, 0x09, 0x09, 0x0E, 0x0B },
+    /* ENKO_TYPE_CHILD_1    */ { 0x02, 0x0C, 0x02, 0x0D, 0x0D },
+    /* ENKO_TYPE_CHILD_2    */ { 0x0B, 0x0B, 0x0B, 0x0F, 0x09 },
+    /* ENKO_TYPE_CHILD_3    */ { 0x00, 0x10, 0x10, 0x11, 0x12 },
+    /* ENKO_TYPE_CHILD_4    */ { 0x13, 0x13, 0x14, 0x0A, 0x09 },
+    /* ENKO_TYPE_CHILD_5    */ { 0x03, 0x03, 0x03, 0x03, 0x03 },
+    /* ENKO_TYPE_CHILD_6    */ { 0x04, 0x16, 0x16, 0x04, 0x17 },
+    /* ENKO_TYPE_CHILD_7    */ { 0x18, 0x10, 0x10, 0x19, 0x10 },
+    /* ENKO_TYPE_CHILD_8    */ { 0x1A, 0x0F, 0x0F, 0x1A, 0x0F },
+    /* ENKO_TYPE_CHILD_9    */ { 0x03, 0x03, 0x03, 0x1B, 0x1B },
+    /* ENKO_TYPE_CHILD_10   */ { 0x02, 0x02, 0x02, 0x02, 0x16 },
+    /* ENKO_TYPE_CHILD_11   */ { 0x0E, 0x0E, 0x0E, 0x0E, 0x0E },
+    /* ENKO_TYPE_CHILD_FADO */ { 0x05, 0x05, 0x05, 0x05, 0x05 }
+};
+
+typedef struct {
+    u8 headId;
+    u8 bodyId;
+    Color_RGBA8 tunicColor;
+    u8 legsId;
+    Color_RGBA8 bootsColor;
+} EnKoModelInfo;
+
+typedef enum { KO_BOY, KO_GIRL, KO_FADO } KokiriGender;
```

Comment:
I suggest including offset comments in this enum to maintain a proper formatting. Also a lot of the other structs/enums declared here could use offset/size comments